src/table/components/TableBodyWrapper.jsx

```diff
@@ -64,13 +45,31 @@ function TableBodyWrapper({
   }, []);
 
   return (
-    <TableBody className={classes.cellBase}>
+    <TableBody
+      sx={{
+        '& td, th': {
+          color: bodyStyle.color,
+          fontSize: bodyStyle.fontSize,
+          padding: bodyStyle.padding,
+        },
+      }}
```

Comment:
We probably want to something similar to makeStyles to put the actual styling somewhere else to not bloat the return

Reply:
Done.

Reply:
It still looks the same to me. I meant like this https://mui.com/system/the-sx-prop/#typescript-usage. ignore the typescript part

Reply:
And that goes for every where we use it, I think it would help a lot with readability